src/ripple/app/main/Main.cpp

```diff
@@ -397,14 +397,12 @@ int run (int argc, char** argv)
             config->VALIDATION_QUORUM = vm["quorum"].as <std::size_t> ();
             if (config->VALIDATION_QUORUM == static_cast <std::size_t>(0))
             {
-                throw std::invalid_argument(
-                        std::to_string(*config->VALIDATION_QUORUM));
+                throw std::domain_error("0");
             }
         }
-        catch(std::exception const&)
+        catch(std::exception const& e)
         {
-            std::cerr << "Invalid quorum = " <<
-                vm["quorum"].as <std::string> () << std::endl;
+            std::cerr << "Invalid quorum = " << e.what() << "\n";;
```

Comment:
Micro-nit: double semi-colon.

I don't think that `e.what()` will actually contain any helpful information here, but consider this format instead:

    std::cerr << "Invalid value specified for --quorum (" << e.what() << ")" << std::endl;

Also, please make the same fix in the previous block (the `catch` that deals with *rpc_port*) since it contains the same bug.